tahoma_description/urdf/pod.xacro

```diff
@@ -249,23 +252,18 @@
         </xacro:if>
         <xacro:unless value="${POD_ID == 1 and i == 0}">
           <xacro:property name="bin_width" value="${BIN_WIDTH}"/>
-          <xacro:property name="NUM_COLUMNS" value="2"/>
+          <xacro:if value="${POD_ID == 1}">
```

Comment:
It's odd for this file to have hardcoded behavior specific to particular pods, since anything that requires hard coding likely could've been included in the recipe as a parameter. I pushed a version of the shelf stack macro which takes the number of columns as a list argument for an idea of how that might work.